src/Security/Authorization/Core/src/DefaultAuthorizationService.cs

```diff
@@ -77,7 +77,7 @@ public DefaultAuthorizationService(IAuthorizationPolicyProvider policyProvider,
         /// A flag indicating whether authorization has succeeded.
         /// This value is <value>true</value> when the user fulfills the policy otherwise <value>false</value>.
         /// </returns>
-        public async Task<AuthorizationResult> AuthorizeAsync(ClaimsPrincipal user, object? resource, IEnumerable<IAuthorizationRequirement> requirements)
+        public virtual async Task<AuthorizationResult> AuthorizeAsync(ClaimsPrincipal user, object? resource, IEnumerable<IAuthorizationRequirement> requirements)
```

Comment:
Is there a reason this couldn't use explicit interface implementation?

Reply:
I believe that way you can only replace the methods instead of building upon them?

When allowing to override, a developer can still call the base method from inside the overridden method to reuse the logic and add own behavior.